packages/devtools_app/lib/src/flutter/app.dart

```diff
@@ -239,6 +233,137 @@ class OpenSettingsAction extends StatelessWidget {
+    )..entering();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final textTheme = Theme.of(context).textTheme;
+
+    return AlertDialog(
+      actions: <Widget>[
+        FlatButton(
+          onPressed: () {
+            Navigator.of(context, rootNavigator: true).pop('dialog');
+          },
+          child: const Text('CLOSE'),
+        ),
+      ],
+      content: Column(
+        mainAxisSize: MainAxisSize.min,
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          Text('DevTools', style: textTheme.headline6),
+          const PaddedDivider(
+            padding: EdgeInsets.only(bottom: denseRowSpacing),
+          ),
+          const SelectableText('DevTools version ${devtools.version}'),
+          //
+          const SizedBox(height: defaultSpacing),
+          Text('Connected Device Info', style: textTheme.headline6),
+          const PaddedDivider(
+              padding: EdgeInsets.only(bottom: denseRowSpacing)),
+          _createDeviceInfo(context, textTheme, _flutterVersion),
+          //
+          const SizedBox(height: defaultSpacing),
+          Text('Feedback', style: textTheme.headline6),
+          const PaddedDivider(
+            padding: EdgeInsets.only(bottom: denseRowSpacing),
+          ),
+          Wrap(
+            children: [
+              const Text('Encountered an issue? Let us know at '),
+              _createFeedbackLink(context, textTheme),
+              const Text('.')
+            ],
+          ),
+        ],
+      ),
+    );
+  }
+
+  Widget _createDeviceInfo(
+    BuildContext context,
+    TextTheme textTheme,
+    FlutterVersion flutterVersion,
+  ) {
+    if (flutterVersion == null) {
+      return const Text('No Flutter device connected.');
+    }
+
+    final versions = {
+      'Flutter': flutterVersion.version,
+      'Framework': flutterVersion.frameworkRevision,
+      'Engine': flutterVersion.engineRevision,
+      'Dart': flutterVersion.dartSdkVersion,
+    };
+
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: <Widget>[
+        for (var name in versions.keys)
+          Padding(
+            padding: const EdgeInsets.only(bottom: 8.0),
+            child: SelectableText('$name ${versions[name]}'),
+          ),
+      ],
+    );
+  }
+
+  Widget _createFeedbackLink(BuildContext context, TextTheme textTheme) {
+    return InkWell(
+      onTap: () async {
+        // TODO(devoncarew): Support analytics.
+        // ga.select(ga.devToolsMain, ga.feedback);
+
+        const reportIssuesUrl = 'https://github.com/flutter/devtools/issues';
+        if (await url_launcher.canLaunch(reportIssuesUrl)) {
+          await url_launcher.launch(reportIssuesUrl);
+        } else {
+          Notifications.of(context).push('Unable to open url.');
+          Notifications.of(context).push('File issues at $reportIssuesUrl.');
+        }
+      },
+      child: Text(
+        'github.com/flutter/devtools/issues',
```

Comment:
maybe pull into a const `reportIssuesUrlShort` or similar. Then above `reportIssuesUrl` could be `'https://$reportIssuesUrlShort'`